frontend/src/components/CaseDetails.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import styles from "src/components/CaseDetails.module.css";
+import Accordion from "@mui/material/Accordion";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import Typography from "@mui/material/Typography";
+import { SingleDetail, DropdownDetail } from "@/components";
+import { useParams } from "next/navigation";
+import { getVSR, type VSR } from "@/api/VSRs";
+
+export const CaseDetails = () => {
+  const [vsr, setVSR] = useState<VSR>({} as VSR);
+  const { id } = useParams();
+
+  useEffect(() => {
+    getVSR(id as string).then((result) => {
+      if (result.success) {
+        setVSR(result.data);
+      }
+    });
+  }, [id]);
+  const expanded = true;
+  return (
+    <div className={styles.box}>
+      <Accordion className={styles.accordian} defaultExpanded expanded={expanded}>
+        <AccordionSummary
+          className={styles.accordianTitle}
+          aria-controls="panel2-content"
+          id="panel2-header"
+        >
+          <Typography className={styles.title}>Case Details</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <div className={styles.details}>
+            <SingleDetail title="Case ID:" value={vsr.caseId} valueFontSize="20px" />
+            <SingleDetail title="Date Received:" value={vsr.dateReceived} valueFontSize="20px" />
```

Comment:
This is what I see - do they look like this on your end?
![image](https://github.com/TritonSE/PAP-Inventory-Processing/assets/25836557/c1f7750e-47d0-42a1-bf1a-e1150cbe9002)

Reply:
Oh I see, I think the dates are formatted differently on my end because they were saved in a different format in my local database, and you're just displaying the string from the database. We should change these fields to be dates instead of strings later, but for now this is fine